subresource-loading.md

```diff
@@ -72,17 +72,25 @@ Content blockers have a number of requirements when it comes to ensuring that ba
 - Resource bundles must not enable the cheap rotation of URLs within the bundle, as this would make URL-based content blocking much more difficult. (c.f. ["Personalization"](https://github.com/littledan/resource-bundles/blob/main/subresource-loading.md#personalization))
 - When content is blocked, it's ideal if sites don't tend to cause browsers to download the blocked content. (c.f. ["code splitting"](#for-web-developers))
 
-## One design: Two-level chunking with client-side hints manifest
+## Proposals
 
-Below is just one concrete place in the design space, as a concrete starting point for discussion and prototyping. There are many possible alternatives to continue considering, discussed in the Design FAQ below.
+Below is a set of potential concrete places in the design space, as concrete starting points for discussion and prototyping. There are many possible alternatives to continue considering, discussed in the Design FAQ below.
+
+### Proposal #1: Two-level chunking with client-side hints manifest
+
+This proposal assumes the following:
+* The site's code is bundled into a single "mega bundle".
+* The bundle declaration on each entry points contains a *complete* list of routes and their dependencies.
```

Comment:
"route" is a little incomplete here, as code splitting may be by components (as @jkrems explained to me, and is used by react-loadable as well).

Reply:
"list of routes/components and their dependencies" ?

Reply:
Why not just "chunks"?

Reply:
At the intro, I prefer to use more descriptive terms